.github/workflows/e2e-sync.yaml

```diff
@@ -18,7 +18,7 @@ jobs:
         uses: engineerd/setup-kind@v0.5.0
         with:
           version: v0.11.1
-          image: kindest/node:v1.21.1@sha256:69860bda5563ac81e3c0057d654b5253219618a22ec3a346306239bba8cfa1a6
+          image: kindest/node:v1.23.13@sha256:ef453bb7c79f0e3caba88d2067d4196f427794086a7d0df8df4f019d5e336b61
```

Comment:
pulled it from here: https://github.com/kubernetes-sigs/kind/releases/tag/v0.17.0